libs/blocks/merch-card/merch-card.js

```diff
@@ -0,0 +1,204 @@
+const PLANS_CARD = 'PlansCard';
+
+const getPodType = (styles) => {
+  const cardTypes = {
+    segment: SEGMENT_BLADE,
+    'special-offers': SPECIAL_OFFERS,
+    plans: PLANS_CARD,
+  };
+  const authoredType = styles?.find((style) => style in cardTypes);
+  return cardTypes[authoredType] || SEGMENT_BLADE;
+};
+
+const createDescription = (rows, cardType) => {
+  const descriptions = rows.slice(0, rows.length - 1);
+  const descriptionWrapper = createTag('div', { class: `consonant-${cardType}-description` });
+  descriptions?.forEach((description) => descriptionWrapper.appendChild(description));
+  return descriptionWrapper;
+};
+
+const createTitle = (titles, cardType) => {
+  const titleWrapper = createTag('div', { class: `consonant-${cardType}-title` });
+  titles?.forEach((title) => titleWrapper.appendChild(title));
+  return titleWrapper;
+};
+
+const decorateFooter = (el, styles, cardType) => {
+  const cardFooter = el.querySelector('.consonant-CardFooter');
+  const replacePlaceHolder = async (key, defaultValue) => {
+    const replacedKey = await replaceKey(key, getConfig(), defaultValue);
+    return { replacedKey } || 'Secure transaction';
+  };
+  const decorateWithSecureTransactionSign = () => {
+    const secureTransactionWrapper = createTag('div', { class: 'secure-transaction-wrapper' });
+    replacePlaceHolder('secure-transaction').then(({ replacedKey }) => {
+      const label = createTag('span', { class: 'secure-transaction-label' }, replacedKey);
+      const secureElement = createTag('span', { class: 'secure-transaction-icon' });
+      secureTransactionWrapper.append(secureElement, label);
+    });
+    return secureTransactionWrapper;
+  };
+
+  const createCheckbox = (checkBoxText) => {
+    cardFooter.querySelector('hr')?.remove();
+    const container = createTag('label', { class: 'checkbox-container' });
+    const input = createTag('input', { id: 'alt-cta', type: 'checkbox' });
+    const checkmark = createTag('span', { class: 'checkmark' });
+    const label = createTag('span', { class: 'checkbox-label' }, checkBoxText.innerHTML);
+    container.append(input, checkmark, label);
+    return container;
+  };
+
+  const decorateAlternativeCta = () => {
+    const toggleButtonActiveState = (buttonToAdd, buttonToRemove) => {
+      buttonToAdd.classList.add('button--inactive');
+      buttonToRemove.classList.remove('button--inactive');
+    };
+    const cardFooterRow = el.querySelector('.consonant-CardFooter-row');
+    if ([...el.classList].includes('secure')) {
+      const standardWrapper = createTag('div', { class: 'standard-wrapper' });
+      const secureTransactionWrapper = decorateWithSecureTransactionSign();
+      standardWrapper.append(secureTransactionWrapper, cardFooterRow);
+      cardFooter?.append(standardWrapper);
+    }
+
+    const altCta = [...el.querySelectorAll('div > div[data-align="center"][data-valign="middle"]')].filter((data) => data.textContent?.trim() === 'alt-cta');
```

Comment:
I feel like this selector is also an issue, same as below. 
My thought is we shouldn't rely on `data-valign` attributes as they can cause an authoring gotcha. Im open, LMK your thoughts.

Reply:
logic updated for altCta and ribbon data search